hazelcast/src/main/java/com/hazelcast/internal/compatibility/cache/CompatibilityCacheDataSerializerHook.java

```diff
@@ -0,0 +1,124 @@
+    public static final short LOAD_ALL_FACTORY = 20;
+    public static final short EXPIRY_POLICY = 21;
+    public static final short KEY_ITERATOR = 22;
+    public static final short KEY_ITERATION_RESULT = 23;
+    public static final short ENTRY_PROCESSOR = 24;
+    public static final short CLEAR_RESPONSE = 25;
+    public static final short CREATE_CONFIG = 26;
+    public static final short GET_CONFIG = 27;
+    public static final short MANAGEMENT_CONFIG = 28;
+    public static final short LISTENER_REGISTRATION = 29;
+    public static final short DESTROY_CACHE = 30;
+    public static final short CACHE_EVENT_DATA = 31;
+    public static final short CACHE_EVENT_DATA_SET = 32;
+    public static final short BACKUP_ENTRY_PROCESSOR = 33;
+    public static final short REMOVE_ALL = 34;
+    public static final short REMOVE_ALL_BACKUP = 35;
+    public static final short REMOVE_ALL_FACTORY = 36;
+    public static final short PUT_ALL = 37;
+    public static final short LEGACY_MERGE = 38;
+    public static final short INVALIDATION_MESSAGE = 39;
+    public static final short BATCH_INVALIDATION_MESSAGE = 40;
+    public static final short ENTRY_ITERATOR = 41;
+    public static final short ENTRY_ITERATION_RESULT = 42;
+    public static final short CACHE_PARTITION_LOST_EVENT_FILTER = 43;
+    public static final short DEFAULT_CACHE_ENTRY_VIEW = 44;
+    public static final short CACHE_REPLICATION = 45;
+    public static final short CACHE_POST_JOIN = 46;
+    public static final short CACHE_DATA_RECORD = 47;
+    public static final short CACHE_OBJECT_RECORD = 48;
+    public static final short CACHE_PARTITION_EVENT_DATA = 49;
+
+    public static final short CACHE_INVALIDATION_METADATA = 50;
+    public static final short CACHE_INVALIDATION_METADATA_RESPONSE = 51;
+    public static final short CACHE_ASSIGN_AND_GET_UUIDS = 52;
+    public static final short CACHE_ASSIGN_AND_GET_UUIDS_FACTORY = 53;
+    public static final short CACHE_NEAR_CACHE_STATE_HOLDER = 54;
+    public static final short CACHE_EVENT_LISTENER_ADAPTOR = 55;
+    public static final short EVENT_JOURNAL_SUBSCRIBE_OPERATION = 56;
+    public static final short EVENT_JOURNAL_READ_OPERATION = 57;
+    public static final short EVENT_JOURNAL_DESERIALIZING_CACHE_EVENT = 58;
+    public static final short EVENT_JOURNAL_INTERNAL_CACHE_EVENT = 59;
+    public static final short EVENT_JOURNAL_READ_RESULT_SET = 60;
+    public static final int PRE_JOIN_CACHE_CONFIG = 61;
+    public static final int CACHE_BROWSER_ENTRY_VIEW = 62;
+    public static final int GET_CACHE_ENTRY_VIEW_PROCESSOR = 63;
+
+    public static final int MERGE_FACTORY = 64;
+    public static final int MERGE = 65;
+    public static final int ADD_CACHE_CONFIG_OPERATION = 66;
+    public static final int SET_EXPIRY_POLICY = 67;
+    public static final int SET_EXPIRY_POLICY_BACKUP = 68;
+    public static final int EXPIRE_BATCH_BACKUP = 69;
+    public static final int CACHE_DESTROY_EVENT_CONTEXT = 70;
```

Comment:
Yeah, I could not decide if I should remove ALL unused constants or leave them, in all of the factories and helper classes. I guess I left them in case I needed them in the future when I would discover I forgot to cover some edge case.

Reply:
Done, removed all unused constants in a separate commit.